Makefile

```diff
@@ -43,7 +43,7 @@ mock: docker-builder
 # Run unit-tests
 unit-test: mock
 	$(DOCKERIZE) go test -coverprofile=coverage-unit.txt \
-		-covermode=atomic -count 1 $$(go list antrea.io/nephe/pkg/...)
+		-covermode=atomic -count 1 $$(go list antrea.io/nephe/pkg/... antrea.io/nephe/apis/crd/v1alpha1/...)
```

Comment:
can we just include files rather than dir. it will expose entire dir with auto generated files and we cannot add coverage for it. Investigate a bit. + @Anandkumar26

Reply:
Discussed offline. 
We need to refactor  the code code, such that unit-test can be added for webhook related files in a separate package.